### PR TITLE
Add/javascript playbook

### PR DESCRIPTION
**Pull request by renatoruk**

### CODE REVIEW

**Comment by renatoruk** [comment]
@tonymrakovcic Thanks for fixing recursive readme! Can we merge this?

File: playbooks/javascript/README.md [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+<!-- prettier-ignore-start -->
+<!-- start_toc -->
+| Title | Overview |
+|---|---|
+| [Automated testing](/playbooks/javascript/automated-testing.md#readme) | Methodologies for integrating tests as a part of the development cycle |
+| [React](/playbooks/javascript/react.md#readme) | Reference book for writing React applications and components |
+| [TypeScript](/playbooks/javascript/typescript.md#readme) | Best practices, rules, guides, and everything in between |
+<!-- end_toc -->
+<!-- prettier-ignore-end -->
```
Comment (degdebug) [inline playbooks/javascript/README.md]: - :warning: You may have missed a TODO here

**Comment by degdebug** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  **playbooks/javascript/README.md#L1** - You may have missed a TODO here
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **playbooks/javascript/README.md#L2** - You may have missed a TODO here
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **playbooks/javascript/README.md#L2** - You may have missed a TODO here
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **playbooks/javascript/README.md#L2** - You may have missed a TODO here
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **playbooks/javascript/README.md#L3** - You may have missed a TODO here
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **playbooks/javascript/README.md#L4** - You may have missed a TODO here
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **playbooks/javascript/README.md#L5** - You may have missed a TODO here
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **playbooks/javascript/README.md#L6** - You may have missed a TODO here
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **playbooks/javascript/README.md#L7** - You may have missed a TODO here
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **playbooks/javascript/README.md#L8** - You may have missed a TODO here
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **playbooks/javascript/README.md#L9** - You may have missed a TODO here
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **playbooks/javascript/README.md#L10** - You may have missed a TODO here
  </td>
    </tr>
  
<tr>
      <td>:warning:</td>
      <td>

  **playbooks/javascript/README.md#L13** - You may have missed a TODO here
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against d430fdd5a789994f707a3b01486cad7f44202d7e
</p>

File: playbooks/javascript/README.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+### [TODO]
+
+<!-- prettier-ignore-start -->
+<!-- start_toc -->
+| Title | Overview |
+|---|---|
+| [Automated testing](/playbooks/javascript/automated-testing.md#readme) | Methodologies for integrating tests as a part of the development cycle |
+| [React](/playbooks/javascript/react.md#readme) | Reference book for writing React applications and components |
+| [TypeScript](/playbooks/javascript/typescript.md#readme) | Best practices, rules, guides, and everything in between |
```
Comment (degdebug) [inline playbooks/javascript/README.md]: - :warning: You may have missed a TODO here

File: playbooks/javascript/README.md [hunk 3]
```diff
@@ -0,0 +1,13 @@
+### [TODO]
+
+<!-- prettier-ignore-start -->
+<!-- start_toc -->
```
Comment (degdebug) [inline playbooks/javascript/README.md]: - :warning: You may have missed a TODO here

File: playbooks/javascript/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
...
+| Title | Overview |
+|---|---|
```
Comment (degdebug) [inline playbooks/javascript/README.md]: - :warning: You may have missed a TODO here

**Comment by tonymrakovcic** [comment]
Oooh nice first time I see danger bot in action :)
Can you just remove the todos and id everything is ok we can merge

**Pull request merged**